apps/src/code-studio/components/header/SignInCallout.jsx

```diff
@@ -110,8 +110,11 @@ const styles = {
     width: 400,
     textAlign: 'left',
     whiteSpace: 'normal',
+    color: 'white',
+    fontWeight: '400',
```

Comment:
I'm not using variables here since they aren't being used in this file and refactoring this further feels out of scope.